node/expressjs/middlewares-in-express-js.md

```diff
@@ -0,0 +1,61 @@
+## Middlewares in Express JS
+
+It's been a long time that NODE JS has become a part of JS Ecosystem. Any JS developer Front End or Full Stack would be familiar with Node JS. Any one who has worked on Node would have also worked with Express JS.
+
+So, Express JS is nothing but a framework to help organize your web application into an MVC architecture on the server side. It basically helps you manage everything from routes to handling requests and views for your web application.
+
+### What are Middlewares
+
+A middleware is just a function which takes request(req), response(res) and the next middleware function(next) as the arguments. These are the function which helps to intercept the incoming request, perform a specific task and pass the execution to next middleware or route handler or even they can end the request-response cycle by returning a response.
```

Comment:
adding a usecase would be better. Like say auth middleware. Tell the user why they should read further